cmd/minishift/cmd/start_preflight.go

```diff
@@ -50,6 +50,10 @@ const (
 	hypervDefaultVirtualSwitchId = "c08cb7b8-9b3c-408e-8e30-5e16a3aeb444"
 )
 
+var (
+	requestedOpenShiftVersion string
+)
+
```

Comment:
Why this is now required since now you are already evaluating it for each function?

Reply:
Leftover of previous implementation :wink: